.env.production

```diff
@@ -0,0 +1,7 @@
+# The port for the development server
+PORT=8080
```

Comment:
That is already in `.env` and also unused in the production build, I think?

Reply:
yap. will be removed